src/registry_schemas/example_data/schema_data.py

```diff
@@ -1217,6 +1217,17 @@
     'courtOrder': COURT_ORDER
 }
 
+CONTINUATION_OUT = {
+    'details': 'A note to explain the continue out',
+    'continuationOutDate': '2023-05-01',
```

Comment:
Or, effective date? Whatever is most consistent with other filings.

Reply:
I did not named it as effective date to avoid confusion with filing effective date